tests/tests/test_filetransfer.py

```diff
@@ -70,15 +74,52 @@ def upload_file(path, file, devid, authtoken, mode="600", uid="0", gid="0"):
     return requests.put(upload_url, verify=False, headers=authtoken, files=files)
 
 
+def set_limits(mender_device, limits):
+    tmpdir = tempfile.mkdtemp()
+    try:
+        # retrieve the original configuration file
+        output = mender_device.run("cat /etc/mender/mender-connect.conf")
+        config = json.loads(output)
+        # replace mender.conf with an mTLS enabled one
+        config["Limits"] = limits
+        mender_connect_conf = os.path.join(tmpdir, "mender-connect.conf")
+        with open(mender_connect_conf, "w") as fd:
+            json.dump(config, fd)
+        mender_device.run(
+            "cp /etc/mender/mender-connect.conf /etc/mender/mender-connect.conf-backup-`ls /etc/mender/mender-connect.* | wc -l`"
+        )
+        mender_device.put(
+            os.path.basename(mender_connect_conf),
+            local_path=os.path.dirname(mender_connect_conf),
+            remote_path="/etc/mender",
+        )
+    finally:
+        shutil.rmtree(tmpdir)
+    connect_service_name = "mender-connect"
+    mender_device.run("systemctl restart %s" % connect_service_name)
+    # mender_device.run("ln -s /etc/mender/mender-connect.conf /etc/mender/mender-connect-link.conf")
+    debugoutput = mender_device.run("cat /etc/mender/mender-connect.conf")
+    logger.info("/etc/mender/mender-connect.conf:\n%s" % debugoutput)
+    debugoutput = mender_device.run("ls -al /etc/mender")
+    logger.info("ls -al /etc/mender/:\n%s" % debugoutput)
+
+
 class _TestFileTransferBase(MenderTesting):
-    def test_filetransfer(self, devid, authtoken):
+    def test_filetransfer(
+        self, devid, authtoken, path="/etc/mender/mender.conf", content_assertion=None
+    ):
         # download a file and check its content
-        path = "/etc/mender/mender.conf"
         r = download_file(path, devid, authtoken)
+
+        logger.info("_TestFileTransferBase got response .json(): %s" % r.json())
+        logger.info("_TestFileTransferBase got response .json(): %s" % r.json())
+        logger.info("_TestFileTransferBase got response .content: %s" % r.content)
```

Comment:
Do you need all three logs? One is enough at most.

Reply:
leftover debug logs. removed.